aws/database-tools/blazer.tf

```
@@ -56,6 +56,9 @@ resource "aws_ecs_task_definition" "blazer" {
       "environment" : [{
         "name" : "LOG_LEVEL",
         "value" : "info"
+        }, {
+        "name" : "NOTIFY_URL",
+        "value" : "${var.notify_url}"
```

Apologies. After my most recent changes this can be switched to:

```suggestion
        "value" : "https://${var.base_url}"
```

We can then switch the variable name in variables.tf to be base_url.

---

Ah I was wondering if we had something like that and couldn't find it. That's good!